src/dlgprefreplaygaindlg.ui

```diff
@@ -202,7 +248,20 @@
       <item row="4" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
-         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not change the ReplayGain of playing tracks.</string>
+         <string>It may take a few seconds after track analysis is finished for a newly calculated ReplayGain to be applied. To avoid abrupt volume changes, Mixxx does not apply the ReplayGain to playing tracks. </string>
+        </property>
+        <property name="wordWrap">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
Sorry for being such un-precise. 

>> He (note @ywwg ) will prefer a boost of 0 dB for both sliders

because Club tracks are already Mixed in an equal loudness, below the one uses for radio tracks. 

>> I guess you mean ReplayGain enabled (otherwise using 2 sliders doesn't make sense).

ReplayGain disabled was correct. Only radio tracks are played much louder than with ReplayGain.
So the default loudness for the majority will be louder than in @ywwg case, it is not possible to increase the loudness of such a club track to radio level without clipping.  

Here some ASCII -Arts: 


original waveform of a club track 
```
1   -    -
   -      
  -  - -- -----
0     -
  -  - - ------
   -         
-1  -   - 
```
original waveform of a radio track 
```
1  -   
  - - -- -----
     -
0       -
      -
  -- - - ------
-1  -         
``` 

Mixing a radio track to a stream of club tracks (@ywwg's case): 
```
               v radio track with pregain = -6 dB 
1   -    -    |      
   -          |  -   
  -  - -- ----| - - -- -----    
0     -       |    -  -
  -  - - -----| --  -- -----     
   -          |   -
-1  -   -     |       
```
Mixing a club track to a stream of radio tracks (my case): 
```
               v club track with pregain = +6 dB 
                  -    -          <- clipping 
1  -          |  -        
  - - -- -----| -  - -- -----
     -        |
0       -     |     -
      -       |
  -- - - -----| -  - - ------
-1  -         |  -    
                  -   -
```
Same case like above (my case), but with default boost = -6 dB (the hidden default in Mixxx 1.11) 
``` 
              v club track with pregain = +6 dB 
			          <- no clipping 
1             |  -    -    
  -           | -        
  - - -- -----|-  - -- ----    
0    -  -     |    -   
  --  -- -----|-  - - -----     
    -         | -          
-1            |  -   -     
```
You see, with the default boost it is possible to tweak the gain in a way that allows to 
keep the pregain at 0dB for the majority of tracks but still allow to mixxx tracks with 
a different (higher) dynamic range. 

This is the feature I do not want to miss in Mixxx 1.12